src/App.tsx

```diff
@@ -21,55 +24,135 @@ enum SortType {
   LENGTH,
 }
 
-// Use this function in the render method
 function getReorderedGoods(
   goods: string[],
   sortType: SortType,
   isReversed: boolean,
 ) {
-  // Not to mutate the original array
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // ...
+  if (sortType !== SortType.NONE) {
+    visibleGoods.sort((good1, good2) => {
+      switch (sortType) {
+        case SortType.ALPABET:
+          return good1.localeCompare(good2);
+        case SortType.LENGTH:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
```

Comment:
It is good when you can avoid nested if, it will be easier to read
```suggestion

  switch (sortType) {
    case SortType.ALPABET:
      return visibleGoods.sort((good1, good2) => good1.localeCompare(good2));
    case SortType.LENGTH:
      return visibleGoods.sort((good1, good2) => good1.length - good2.length);
    case SortType.NONE:
    default:
      break;
  }
```